main.cpp

```diff
@@ -127,393 +163,337 @@ struct CarWash
     int switchChannel(int currentChannel, int newChannel = 0);
 };
-/*
-Thing 4) Power strip
-5 properties:
-    1) Number of power outlets (int)
-    2) max surge  protection rating in watts (float)
-    3) connector format (std::string) 
-    4) wire gauge (int)
-    5) number of populated outlets (int)
-3 things it can do:
-    1) Install plug
-    2) get current outlet power 
-    3) disable power
- */
+
+void GuitarAmp::PowerAmp::changeGain(double gainAdjustment) {gain *= gainAdjustment;}
+void GuitarAmp::PowerAmp::enableAmpStandby() { powerState = "standby";}
+int GuitarAmp::PowerAmp::selectImpedance( int impedance) 
+{ 
+    outputImpedance = impedance;
+    return impedance;
+}
+
+double GuitarAmp::amplifyGuitar(double inputVoltage, double ampGain) 
+{ 
+    return inputVoltage * ampGain;
+}
+
+double GuitarAmp::adjustReverbAmount(double reverbGain)
+{
+    reverbLevel *= reverbGain;
+    return reverbLevel;
+}
+
+int GuitarAmp::switchChannel(int currentChannel, int newChannel)
+{
+    if (currentChannel == newChannel) return currentChannel;
+
+    return newChannel;
+}
 
 struct PowerStrip
 {
-    // Number of power outlets 
-    int numOutlets = 5;
-    // max surge  protection rating in watts
+    int numOutlets = 3;
     float maxSurgeProtection = 2400.0f;
-    //  
-    
-    // wire gauge 
     int wireGauge = 18;
-    // number of populated outlets 
     int numPopulatedOutlets = 2;
 
     struct Outlet
     {
-        // which outlet number
         int outletNum = 1;
-        // current power
-        double currentPower = 0.0;
-        // GFCI status
-        std::string gfciState = "normal";
-        // Connector format
+        float currentPower = 0.0;
+        std::string GFCIState = "normal";
         std::string connectorFormat = "US";
-        // plug installed
         bool plugInstalled = false;
 
-        // Trip GFCI breaker
         void tripGFCI();
-        // retrieve outlet current power
-        double getCurrentPower();
-        // return plugged in status (useful for parent functions)
+        float getCurrentPower();
         bool isPlugInstalled();
-
     };
 
-    // Insert plug
+    Outlet outlet1;
+    Outlet outlet2;
+    Outlet outlet3;
+
     int insertPlug(Outlet outlet);
-    // get current outlet power
-    float getOutletPower(int plugNum);
-    // disable power
+    float getOutletPower();
     void disablePower();
-      
-
 };
-/*
-Thing 5) Pickups
-5 properties:
-    1) Magnet Material (std::string)
-    2) Number of Coils (int)
-    3) type of pickup
-    4) distance from strings in cm (float)
-    5) brand (std::string)
-3 things it can do:
-    1) adjust height
-    2) Select tap
-    3) convert vibration to voltage
- */
+
+void PowerStrip::Outlet::tripGFCI()
+        {
+            currentPower = 0.0;
+            GFCIState = "tripped";
+        }
+float PowerStrip::Outlet::getCurrentPower() { return currentPower;}
+bool PowerStrip::Outlet::isPlugInstalled() { return plugInstalled;}
```

Comment:
If you write single-line functions, make sure you have space around the `{ }` so the code is easier to read
- [ ] revise